swagger-spec/identifiers/definition.yaml

```diff
@@ -0,0 +1,47 @@
+type: object
+title: Identifier
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: 'The identifier of the identifier entity.'
+  type:
+    type: string
+    readOnly: true
+    description: 'The type identifier of the identifier entity (`identifiers`).'
+  attributes:
+    type: object
+    title: Attributes
+    readOnly: true
+    description: 'The properties of the identifier entity.'
+    properties:
+      category:
+        type: string
+        readOnly: true
+        description: 'category can be ark or doi'
```

Comment:
"The category of the identifier."

You can use the `enum_fields` key to specify that the accepted categories are ARK and DOI. It's used in the [node definition](https://github.com/CenterForOpenScience/developer.osf.io/blob/develop/swagger-spec/nodes/definition.yaml#L29) for the category field as well.